tests/bootstrap.php

```diff
@@ -44,8 +45,8 @@
     ],
     'postgres'  => [
         'driver'     => Database\Driver\Postgres\PostgresDriver::class,
-        'check'      => static function () {
-            return !in_array('pgsql', \PDO::getAvailableDrivers(), true);
+        'check'      => static function () use ($db) {
+            return $db === 'postgres' || in_array('pgsql', \PDO::getAvailableDrivers(), true);
         },
```

Comment:
I'm not sure that these checks are used somewhere. Mb it should be removed?

Reply:
Yes, they can. Sounds like they are redundant.